### DISCUSSION TITLE
Full compatibility with MATEKH743

### DISCUSSION THREAD

**Discussion by tvquinn** [iNavFlight/inav#7436] [Q&A] [answered]
Hi,

I am currently in the process of converting my Quad using the MATEKH743 Flight Controller from Betaflight to iNAV and notices that the MATEKH743 was not fully supported in iNAV (e.g. Accelerometer, Gyro and other items are not fully supported in the latest version of the Configurator. I believe the MATEKH743 includes all the necessary on-board hardware required to fully implement iNAV. 

**### Is there an ETA when the MATEKH743 will be fully supported?**

This notice was in the release page for version INAV 3.0.1 
=====================================
**MATEKH743 support**
This release supports Matek H743-WING, H743-Slim, and H743-Mini flight controllers. Targets are fully flyable, but not all features are yet working:

1) No SD card support and no MSC mode (Blackbox will not work)
2) CAN bus is not supported (

**Comment by packetkiller** [comment]
Any update / eta on when blackbox for Matek H743 will be added to iNAV 4.1?

**Comment by DzikuVx** [comment, marked as answer]
It's working since 4.0 so I'm not sure what else to add here

**Comment by packetkiller** [comment]
The access to the blackbox not working on Lumenier LUX H7 HD.  The blackbox records the data but you can not use the USB connection to get your data. You must remove the SD card and upload the files to a PC.  Yes, this did work in 4.0, thus why I am asking what is going on.All help is appreciated.BTW, I have multiple of these FC and willing to test and send what ever specifics you need to reproduce the issue.V/R
    On Tuesday, April 12, 2022, 12:57:16 AM CDT, Paweł Spychalski ***@***.***> wrote:  
 
 


It's working since 4.0 so I'm not sure what else to add here

**Comment by jvdejesus05** [comment]
Greetings!  I'm planning to buy the Matek H743-Slim V1.5 FC to use with INAV.  I just need to be sure that I can flashed with INAV?

**Comment by DzikuVx** [comment]
Works just fine, I have 2 birds with this FC

**Reply by Antipilas** [reply]
Sorry but my X8 ( I send you a image of it some weeks ago) has mounted this FC and I tried with two different Micro SD of 8 and 16 GB and it does not work....much worse... with SD connected is imposible to connect INAV ...is like the SD is using the communication UART.

**Reply by Antipilas** [reply]
Inav 4.1 is installed

**Comment by packetkiller** [comment]
Are you saying that the "Lumenier LUX H7 HD" running iNAV 4.0.1 using the blackbox feature allowing you to connect to the FC via USB and download your log files is working?
I have several of these are can get the blackbox data by physically removing the SD card and copying/moving the files to PC but am NOT able to use USB to get the log files.
I am attempting to validate this is still a bug or if there is something else going on.
Thank you,

    On Sunday, April 24, 2022, 07:51:24 AM CDT, Paweł Spychalski ***@***.***> wrote:  
 
 


Works just fine, I have 2 birds with this FC
